webpack.prod.js

```diff
@@ -15,8 +16,8 @@ module.exports = {
                 loader: 'babel-loader'
             },
             {
-                test: /\.scss$/,
-                use: [ 'style-loader', 'css-loader', 'sass-loader' ]
+                test: /\.s[ac]ss$/i,
```

Comment:
This is fine as it is still completely correct, but just as a small note, this will match not only `*.scss` files but `*.SCSS` or `*.sass` as well.